src/pages/Trade/components/History/hooks.ts

```diff
@@ -234,6 +234,9 @@ export const usePollOrderList = ({
                 parsed[i].createdAt = blockHeader.timestamp
               }
               const { eth_to_ckb, ckb_to_eth } = (await getForceBridgeHistory(ckbAddress, ethAddress)).data
+              if (abortController.current?.signal.aborted) {
```

Comment:
Wouldn't it be more appropriate to check the `connectStatus !== 'connectted'` of the wallet here rather than using AbortController?